apps/src/templates/studioHomepages/CourseBlocks.jsx

```diff
@@ -4,34 +4,35 @@ import ReactDOM from 'react-dom';
 import ContentContainer from '../ContentContainer';
 import CourseBlocksTools from './CourseBlocksTools';
 import CourseBlocksInternationalGradeBands from './CourseBlocksInternationalGradeBands';
+import {NotificationResponsive} from '@cdo/apps/templates/Notification';
 import ProtectedStatefulDiv from '../ProtectedStatefulDiv';
 import i18n from "@cdo/locale";
 import {pegasus} from '@cdo/apps/lib/util/urlHelpers';
 
 export class CourseBlocksCsf extends Component {
   static propTypes = {
-    isEnglish: PropTypes.bool.isRequired
+    showModern: PropTypes.bool.isRequired
   };
 
   render() {
-    if (this.props.isEnglish) {
-      return (<CourseBlocksCsfEnglish/>);
+    if (this.props.showModern) {
+      return (<CourseBlocksCsfModern/>);
     } else {
-      return (<CourseBlocksCsfNonEnglish/>);
+      return (<CourseBlocksCsfLegacy/>);
     }
   }
 }
 
-class CourseBlocksCsfEnglish extends Component {
+class CourseBlocksCsfModern extends Component {
   componentDidMount() {
-    $('#coursea').appendTo(ReactDOM.findDOMNode(this.refs.coursea)).show();
-    $('#courseb').appendTo(ReactDOM.findDOMNode(this.refs.courseb)).show();
-    $('#coursec').appendTo(ReactDOM.findDOMNode(this.refs.coursec)).show();
-    $('#coursed').appendTo(ReactDOM.findDOMNode(this.refs.coursed)).show();
-    $('#coursee').appendTo(ReactDOM.findDOMNode(this.refs.coursee)).show();
-    $('#coursef').appendTo(ReactDOM.findDOMNode(this.refs.coursef)).show();
-    $('#pre-express').appendTo(ReactDOM.findDOMNode(this.refs.pre_express)).show();
-    $('#express').appendTo(ReactDOM.findDOMNode(this.refs.express)).show();
+    $('#coursea-2017').appendTo(ReactDOM.findDOMNode(this.refs.coursea)).show();
```

Comment:
Separate from this PR, but I'm curious, with course versioning are we going to have to track and update everywhere that we reference a specific year and change it the following year? That sounds potentially cumbersome...

Reply:
Great question for @tanyaparker.  Not sure whether these are expected to change annually or not.

That said, I should be at least using the blocks for the 2018 versions, in particular for the strings we use.  I'll look into fixing that now.

Reply:
Regarding my second point, we are actually translating the 2017 courses and so they're the ones we want to use for now.

Reply:
I think of this as a transition period where we have both 1-4 and A-F. But once we have A-F translated in the majority of languages, we switch to always showing A-F. Then the logic changes to checking for the latest version that has all courses translated. Something along those lines so we don't have to update each year.

Reply:
> Separate from this PR, but I'm curious, with course versioning are we going to have to track and update everywhere that we reference a specific year and change it the following year? That sounds potentially cumbersome...

I agree with the wisdom of past Erin, versioning is indeed cumbersome.

If I'm reading this correctly, there is versioning work to be done to get /courses up to date.  It looks like right now we live in world where "modern elementary courses" refers to the 2017 version of CSF 
https://github.com/code-dot-org/code-dot-org/blob/fa36c0269f37b4dce59f1fe42cc5a1992fdf1a5d/dashboard/app/models/script.rb#L271
And that we explicitly use 2017 on /courses
https://github.com/code-dot-org/code-dot-org/blob/ecdbc30e18d75c522f4450351875851efb8be3ee/apps/src/templates/studioHomepages/CourseBlocks.jsx#L29
But I'm under the impression that we have more recent versions of CSF translated into at least some languages. For example: 
https://github.com/code-dot-org/code-dot-org/blob/423000cc4457e8f6e1b9b8e5084da3d97af483ff/dashboard/config/scripts/coursea-2018.script#L12

@elijah or @nkiruka can I get an overview of the current progress of translations for CSF; we have 2018 versions of CSF translated into some non-en locales, right? 

 @madelynkasula and @davidsbailey  in the work that you've already done for redirects, don't we have logic somewhere that checks for the most recent stable version of a course translated in a specific language? We might want to use that same logic to determine which courses to promote on /courses?

Reply:
no, since courses don't currently have `supported_locales` the way that scripts do, we don't have logic that checks for the most recent stable version in a specific locale. we do have [`Course.latest_stable_version`](https://github.com/code-dot-org/code-dot-org/blob/873c2814feaca60f0819ec24fb98f7de2a0e4052/dashboard/app/models/course.rb#L414-L425), which gives you the latest version in a course family

Reply:
i recently added [LP-377](https://codedotorg.atlassian.net/browse/LP-377) to add `supported_locales` to the Course model, but it hasn't been high enough priority since CSD and CSP aren't translated at all, and all CSF "courses" are scripts.

if we'd only be checking scripts here, we could use [`Script.latest_stable_version`](https://github.com/code-dot-org/code-dot-org/blob/78379591440d6aba4235e523ed003943ce93b5c9/dashboard/app/models/script.rb#L575-L595), which does give you the latest stable version for a locale

Reply:
Got it! Thanks for clarifying @madelynkasula That makes sense and looks like /courses can actually stay as is for now; I just got paranoid we might be missing something 🙃